sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -16,7 +16,7 @@
 
 from __future__ import absolute_import
```

Comment:
Can you please remove this as this repo seems to run the latest lint checks?

Reply:
It was tough, but I got there: https://github.com/shadow-robot/common_resources/pull/246/commits/92687938fefbbd8f2d7bf21b9e332c4e0a6ad0ac